libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/Route.kt

```diff
@@ -1,14 +1,16 @@
 package com.mapbox.navigation.base.route.model
 
-import com.mapbox.api.directions.v5.models.RouteLeg
-
 data class Route(
     val routeIndex: String?,
     val distance: Double?,
     val duration: Long?,
     val geometry: String?,
     val weight: Double?,
     val weightName: String?,
-    val legs: List<RouteLeg>?,
+    val legs: RouteLegsNavigation?,
     val voiceLanguage: String?
 )
+
+data class RouteLegsNavigation(
```

Comment:
Do we want to extract this into a separate file `RouteLegsNavigation.kt`?

Reply:
moved to separate file